src/main/java/com/gmail/goosius/siegewar/utils/FileMgmt.java

```diff
@@ -41,7 +41,7 @@ public static String convertFileToString(File file) {
 				}
 				reader.close();
 			} catch (IOException e) {
-				System.out.println("Exception ");
+				e.printStackTrace();
```

Comment:
:question: 
- Is e.printStackTrace() effectively a System.out.println() ?
- If so, then do these items throughout the code all need to be replaced by some logging call also ?

Reply:
I don't believe these are an issue in their mind. They are trying to get people to stop using println's (because they should only be used for debugging or something.) They consider it a bad coding practice.